External/FEXCore/include/FEXCore/Debug/InternalThreadState.h

```diff
@@ -72,7 +73,7 @@ namespace FEXCore::Core {
   };
 
   struct InternalThreadState {
-    FEXCore::Core::CpuStateFrame* CurrentFrame = &BaseFrameState;
+    FEXCore::Core::CpuStateFrame* const CurrentFrame = &BaseFrameState;
```

Comment:
Switching frames is broken, and not used. Made this const here so we don't accidentally do it. We might have to come back to this and re-merge Thread/Frame.